modules/ROOT/pages/getting-started/toolbox.adoc

```diff
@@ -1,38 +1,39 @@
 [[Toolbox]]
-
+:description: This page describes the functionalities of the Neo4j GraphQL Toolbox.
 = Neo4j GraphQL Toolbox
 
-Experiment with your Neo4j GraphQL API using the purpose-built web IDE.
-
-> This tool is for development and experimentation purposes only!
-
-== Link
-
-Access the Neo4j GraphQL Toolbox under https://graphql-toolbox.neo4j.io
-
-== Usage
-
-1. Connect to a Neo4j database with valid credentials
-2. Define (or introspect) the type definitions
-3. Build the Neo4j GraphQL schema
-4. Experiment, query and play
+The https://graphql-toolbox.neo4j.io[Neo4j GraphQL Toolbox] is an onboarding, low-code tool that can be integrated to Neo4j.
+It was created for development and experimentation with Neo4j GraphQL APIs.
+With it, you can:
 
-== Requirements
+. Connect to a Neo4j database with valid credentials.
+. Define (or introspect) the type definitions.
+. Build the Neo4j GraphQL schema.
+. Experiment, query, and play.
 
-1. A Neo4j database, see https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or use a https://neo4j.com/cloud/[Neo4j AuraDB]
+== Connect to a Neo4j database
 
-2. Make sure the Neo4j database fulfills xref::index.adoc#introduction-requirements[these requirements], including the necessary plugins
+Before start using the Toolbox, make sure you have followed all xref:index.adoc#requirements[requirements] to run a Neo4j database.
+You can use https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or https://neo4j.com/docs/aura/auradb/[Neo4j AuraDB] for that.
 
-== Example
+== Set the type definitions
 
-Define your type definitions directly in the editor or introspect the Neo4j database you connected to.
+. Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
++
+If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see something like this:
++
+image::toolbox-schema-view.png[]
 
-image::toolbox-schema-view-v2.png[title="Type definitions view"]
+. Click the button "Build schema" and then go to the Query editor tab. 
 
-Then query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library.
+. Query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library:
++
+image::toolbox-editor-view.png[]
 
-image::toolbox-editor-view-v2.png[title="Query editor view"]
+== Store type definitions as favorite
 
-Note that you can store your type definitions as favorites and that a range of settings are available to work with.
+You can store your type definitions as favorites and that a range of settings are available to work with.
```

Comment:
This sentence seems wrong now, so it probably needs to be reworded.

Reply:
I don't understand why removing "note that" makes the sentence wrong? Could you please suggest another way or should we revert to its original?